docs/pages/en-us/Architecture.md

```diff
@@ -100,6 +99,8 @@ When the job tracking worker receives a success message to be proxied (that is,
 
 The job tracking worker recognizes failure and retry messages, which are being proxied, and updates the job database accordingly.
 
+The job tracking worker can also automatically forward on dependent jobs for execution.  A dependent job is a job which must wait until a specific job or list of jobs have completed before it can be executed.  The job tracking worker monitors a job's progress, when a job completes, the job tracking worker will receive a list of jobs which can now be executed as the job(s) which these jobs were dependent upon have now completed.
```

Comment:
The last sentence in this paragraph doesn't read well.

Reply:
Will update to:
`The job tracking worker monitors a job's progress, when a job completes, the job tracking worker will receive a list of jobs which can now be executed.`